<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QFrame" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1198</width>
    <height>1260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Load">
      <attribute name="title">
       <string>Load IBL</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>471</width>
         <height>800</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>1200</width>
         <height>1000</height>
        </size>
       </property>
       <property name="title">
        <string>Browse</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="btnBrowse">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>34</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Choose ibl Folder</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>220</x>
             <y>10</y>
             <width>161</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Existing Library</string>
           </property>
          </widget>
          <widget class="QComboBox" name="comboBox">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>200</width>
             <height>27</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listSiblFiles">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1000</width>
            <height>1000</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="isWrapping" stdset="0">
           <bool>false</bool>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>10</y>
         <width>650</width>
         <height>800</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>650</width>
         <height>400</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>650</width>
         <height>800</height>
        </size>
       </property>
       <property name="title">
        <string>Control</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string/>
          </property>
          <widget class="QComboBox" name="comboBoxRenderer">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>20</y>
             <width>200</width>
             <height>27</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string extracomment="Select a renderer"/>
           </property>
           <property name="editable">
            <bool>false</bool>
           </property>
           <property name="maxCount">
            <number>10</number>
           </property>
           <property name="minimumContentsLength">
            <number>3</number>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>220</x>
             <y>20</y>
             <width>171</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Choose Renderer</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnLoad">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>34</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Load ibl</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnUnload">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>34</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Unload ibl</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>600</width>
            <height>300</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>600</width>
            <height>300</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Select a file on the left to preview</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>800</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="nameLabel">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Name</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="locationLabel">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Location</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="commentLabel">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Comment</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="Edit">
      <attribute name="title">
       <string>Edit HDR</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>9</y>
         <width>261</width>
         <height>441</height>
        </rect>
       </property>
       <property name="title">
        <string>Load HDR</string>
       </property>
       <widget class="QPushButton" name="btnLoadHDR">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>200</width>
          <height>24</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>34</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Choose HDR File</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>10</y>
         <width>261</width>
         <height>441</height>
        </rect>
       </property>
       <property name="title">
        <string>Edit HDR</string>
       </property>
       <widget class="QPushButton" name="btnLoadHDR_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>200</width>
          <height>24</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>34</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Create Light</string>
        </property>
       </widget>
       <widget class="QListView" name="listView">
        <property name="geometry">
         <rect>
          <x>15</x>
          <y>70</y>
          <width>231</width>
          <height>111</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>15</x>
          <y>50</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Existing Lights</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
